src/App.js

```diff
@@ -99,11 +100,12 @@ export default function App() {
 
     }, [])
 
-    async function postRequest(inputValue, inputLanguageValue, outputLanguageValue ){
+    async function postRequest(inputValue, inputLanguageValue, outputLanguageValue ){ 
         const params = new URLSearchParams({
           input: inputValue,
           in_lang: inputLanguageValue,
-          out_lang: outputLanguageValue
+          out_lang: outputLanguageValue,
+          id: sessionStorage.getItem('id')
```

Comment:
Would prefer if the `id` string to be extracted into a constant to avoid potential typo issues.

Even better IMO would be to extract the session management into its own file/module and remove the details from the UI part of the code. This would also make it easier to have the ID generated/stored only when it's needed so you can make sure it's never null, which is now only true because of some of the other unrelated state/UI logic.